### PR TITLE
chore(theme): export `Theme` type

### PR DESCRIPTION
**Pull request by ogonkov**

### CODE REVIEW

File: src/components/theme/index.ts [hunk 1]
```diff
@@ -5,3 +5,4 @@ export * from './useTheme';
 export * from './useThemeValue';
 export * from './withTheme';
 export * from './withThemeValue';
+export {Theme as ThemeType} from './types';
```
Comment (amje) [inline src/components/theme/index.ts:8]: Why alias?

Comment (ogonkov) [reply src/components/theme/index.ts:8]:
```js
import {Theme} from '@yandex-cloud/uikit'
```

I think it looks like we import some component

Comment (amje) [reply src/components/theme/index.ts:8]: We already have similar exported types (i.e. `Lang`). No need to add *Type

Comment (ogonkov) [reply src/components/theme/index.ts]: Fixed

**Review by amje — APPROVED** [review]

**Pull request merged**